.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     - uses: actions/checkout@v3
     
     - name: Install Dependencies
-      run: sudo apt install -y meson ninja-build gstreamer1.0-tools gstreamer1.0-plugins-base
+      run: sudo apt install -y meson ninja-build gstreamer1.0-tools libgstreamer1.0-dev
```

Comment:
What's your intention with this workflow?

Reply:
To compile and test each commit.